packages/wdio-browserstack-service/src/Percy/Percy.ts

```diff
@@ -114,20 +123,32 @@ class Percy {
 
     async fetchPercyToken() {
         const projectName = this.#projectName
-
         try {
             const type = this.#isApp ? 'app' : 'automate'
-            const response = await nodeRequest(
-                'GET',
-                `api/app_percy/get_project_token?name=${projectName}&type=${type}`,
+            let query = 'api/app_percy/get_project_token?'
+            if (projectName) {
+                query += `name=${projectName}&`
```

Comment:
Instead of computing the url string like this let's use `URLSearchParams`

Reply:
Sure, thanks for suggesting this